js/app/services/UserService.js

```diff
@@ -0,0 +1,13 @@
+function UserService($http) {
+	this.getUser = function () {
```

Comment:
I'm unclear on how to refactor this using `$resource`. My thinking would be something like this: 

``` js
function UserService($resource) {
  return $resource('/rest/user/:userId')
}
```

But this wont attach a method called  `getUser` - should I attach those properties to the UserService still?